src/terrain/terrain_generation/worley_noise.cpp

```diff
@@ -0,0 +1,81 @@
+#include "worley_noise.hpp"
+
+namespace terrain {
+
+namespace terrain_generation {
+
+WorleyNoise::WorleyNoise(double tile_size = 1) : tile_size_(tile_size) {}
```

Comment:
default args go in the header

and what is the point of such a small func in the c++ file